include/envoy/buffer/buffer.h

```diff
@@ -74,7 +74,7 @@ class Instance {
    * Copy a string into the buffer.
    * @param data supplies the string to copy.
    */
-  virtual void add(const std::string& data) PURE;
```

Comment:
I think we can remove `add(const void* data, uint64_t size)` variant with this, just leave a TODO?

Reply:
Yes, I'll add TODOs. Too many call sites for this PR to cleanup (> 100).